.github/workflows/build_and_test.yml

```diff
@@ -32,31 +35,31 @@ jobs:
         fetch-depth: '0'
     - run: git fetch --depth=1 origin +refs/tags/*:refs/tags/* +refs/heads/*:refs/remotes/origin/*
     - name: Set up Python 3.9
-      if: github.event_name == 'pull_request' || github.event_name == 'release' || contains(github.ref, 'refs/heads/wheel') || github.event_name == 'push'
+      if: github.event_name == 'pull_request' || github.event_name == 'release' || contains(github.ref, 'refs/heads/wheel') || github.event_name == 'schedule'
       uses: actions/setup-python@v3
       with:
         python-version: '3.9'
     - name: Build and test including remote checks (3.9) mypy
-      if:  (matrix.os == 'macos-11') && ((github.event_name == 'pull_request' &&  github.event.pull_request.head.repo.full_name == github.repository) || github.event_name == 'release' || contains(github.ref, 'refs/heads/wheel') || github.event_name == 'push')
+      if:  (matrix.os == 'macos-11') && ((github.event_name == 'pull_request' &&  github.event.pull_request.head.repo.full_name == github.repository) || github.event_name == 'release' || contains(github.ref, 'refs/heads/wheel') || github.event_name == 'push'  || github.event_name == 'schedule')
```

Comment:
I am not sure why this check is run on `push`, too. (other then for all other extensions...

Reply:
This is legacy, because it used to only work on 3.9.

Reply:
Will make a note to update this in the future.